lib/web_ui/lib/src/engine/vector_math.dart

```diff
@@ -373,7 +373,9 @@ class Matrix4 {
       return scaled(arg);
     }
     if (arg is Vector3) {
-      return transformed3(arg);
```

Comment:
aim was to to reduce allocation/copy, why the change?

Reply:
The copyless version is `transform3`. I removed `transformed3`, which was making copies. I wasn't sure if `operator *` is still used, so I'm not changing it yet.